# HG changeset patch
# User John Donoghue <john.donoghue@ieee.org>
# Date 1368968728 14400
#      Sun May 19 09:05:28 2013 -0400
# Node ID c6c4847aaf67228f1437189c5b16c9b7ef16b84d
# Parent  5c5c46a81ad51109ce32da2bc24027c4463bbb61
Make open file "file not found" message box open in front of editor window.

libgui/src/m-editor/file-editor.cc
  (file_editor::request_open_file): set parent of QMessageDialogBox from 0 to the file_editor.

diff --git a/libgui/src/m-editor/file-editor.cc b/libgui/src/m-editor/file-editor.cc
--- a/libgui/src/m-editor/file-editor.cc
+++ b/libgui/src/m-editor/file-editor.cc
@@ -301,17 +301,17 @@ file_editor::request_open_file (const QS
                 {
                   delete fileEditorTab;
                   // Create a NonModal message about error.
                   QMessageBox *msgBox
                     = new QMessageBox (QMessageBox::Critical,
                                        tr ("Octave Editor"),
                                        tr ("Could not open file %1 for read:\n%2.").
                                        arg (openFileName).arg (result),
-                                       QMessageBox::Ok, 0);
+                                       QMessageBox::Ok, this);
 
                   msgBox->setWindowModality (Qt::NonModal);
                   msgBox->setAttribute (Qt::WA_DeleteOnClose);
                   msgBox->show ();
                 }
             }
 
           // really show editor and the current editor tab
